testing.go

```diff
@@ -411,17 +416,18 @@ func (c *cluster) GetInState(s RaftState) []*Raft {
 			c.t.FailNow()
 
 		case <-limitCh:
-			c.FailNowf("[ERR] Timeout waiting for stable %s state", s)
+			c.FailNowf("timeout waiting for stable %s state", s)
 
 		case <-c.WaitEventChan(filter, 0):
-			c.logger.Printf("[DEBUG] Resetting stability timeout")
+			c.logger.Debug("resetting stability timeout")
 
 		case t, ok := <-timer.C:
 			if !ok {
-				c.FailNowf("[ERR] Timer channel errored")
+				c.FailNowf("timer channel errored")
 			}
-			c.logger.Printf("[INFO] Stable state for %s reached at %s (%d nodes), %s from start of poll, %s from cluster start. Timeout at %s, %s after stability",
-				s, inStateTime, len(inState), inStateTime.Sub(pollStartTime), inStateTime.Sub(c.startTime), t, t.Sub(inStateTime))
+
+			c.logger.Info(fmt.Sprintf("stable state for %s reached at %s (%d nodes), %s from start of poll, %s from cluster start. Timeout at %s, %s after stability",
```

Comment:
This one is kind of a tricky one to replace and is only used for testing so I'm going to leave it in.

Reply:
@s-christoff are you alright with leaving this one in?

Reply:
Yeah! LGTM :)